src/util/submit-addon.js

```diff
@@ -89,7 +88,9 @@ export default class Client {
   }
 
   fileFromSync(path) {
-    return fileFromSync(path);
+    // create a blob from a file path
+    const file = readFileSync(path);
+    return new Blob([file]);
   }
```

Comment:
That worked on first paste! 🙌 Which is great because I was ready to throw in the towel 😅 

[![](https://github.com/user-attachments/assets/0bccc76d-6861-4b52-9fb1-47293aa32a30)](https://www.youtube.com/watch?v=g-nmx5iw9oY)